### PR TITLE
adding 2 provdiers I tested

### PR DESCRIPTION
**Pull request by cekvenich**
I tested Wasabit and Digital Ocean. They work! With the -endpoint flag.

### CODE REVIEW

**Comment by cekvenich** [comment]
Fixed order

File: README.md [hunk 1]
```diff
@@ -124,11 +124,14 @@ goofys has been tested with the following non-AWS providers:
 
 * Amplidata
 * DreamObjects (Ceph)
+* Digital Ocean Spaces (Ceph)
```
[the remark this answers is not in the document]
Comment (cekvenich) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -124,11 +124,14 @@ goofys has been tested with the following non-AWS providers:
...
 * EMC Atmos
 * Google Cloud Storage
 * OpenStack Swift
 * S3Proxy
 * Minio (limited)
+* Wasabi 
+
```
[the remark this answers is not in the document]
Comment (cekvenich) [reply README.md]: fixed

**Comment by cekvenich** [comment]
I don't know how to squash commits. (also I'm not using CLI, I'm using the GUI Github desktop)

**Comment by kahing** [comment]
Thanks for the update @cekvenich

**Comment by cekvenich** [comment]
So happy I could contribute a little. I'll try to do better if I can next time.

**Pull request merged**